docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -0,0 +1,58 @@
+# Creating a new network from the Cleura Cloud Control Panel
+
+Before creating a server in {{extra.brand}} Cloud, you need to have a 
+network to make the new server a member of. You may have more than one 
+network per region, so let us now walk through creating a new network 
+using the {{extra.brand}} Cloud Control Panel. Fire up your favorite web 
+browser, navigate to the [Cleura Cloud](https://cleura.cloud) page, and 
+login into your {{extra.brand}} account.
+
+On the top right-hand side of the {{extra.brand}} Cloud Control Panel, 
+click the _Create_ button. A new pane will slide into from the 
+right-hand side of the browser window, titled ---unsurprisingly 
+enough--- _Create_.
+
+![Create a new object](assets/new-net-panel/shot-01.png)
+
+You will notice several rounded boxes prominently displayed on that 
+pane, each for defining, configuring, and instantiating a different 
+{{extra.brand}} Cloud object. Go ahead and click the _Network_ box. A 
+new pane titled _Create Network_ will slide over. At the top, type in a 
+name and select one of the available regions for the new network.
+
+![New network name and region](assets/new-net-panel/shot-02.png)
+
+Expand the _Advanced Options_ section below and make sure _Port 
+Security_ is enabled. You do not have to set a value for the MTU 
+parameter.
+
+![MTU and port security](assets/new-net-panel/shot-03.png)
+
+You probably want a full-featured network for your cloud servers, so 
+unless you know what you are doing, please activate the _Create a 
```

Comment:
Remove the "so unless you know what you are doing, please" bit.

Reply:
Removed it, but this phrase in general doesn't hurt IMHO.

Reply:
I think it really does. "Unless" implies a negative. If I say "unless you have a death wish, you shouldn't ride your bicycle in rush-hour traffic in Athens" implies that the reasonable assumption is that you _don't_ have a death wish. 

Writing "unless you know what you are doing" thus implies that we default to the assumption that the reader does not know what they're doing, and that's a rude assumption.